test/src/e2e_vm_tests/harness.rs

```diff
@@ -52,6 +52,7 @@ pub(crate) fn runs_on_node(file_name: &str) {
         print_ir: false,
         silent_mode: true,
         pretty_print: false,
+        contract: None,
```

Comment:
in the beginning of this function you'll want to pass in the contracts as arguments

Reply:
Then in the `mod.rs`, sibling to this file, pass them in. That's where youll do the calculation and handling